content/docs/thinking-in-react.md

```diff
@@ -1,24 +1,24 @@
 ---
 id: thinking-in-react
-title: Thinking in React
+title: Cara Berpikir Dengan React
 permalink: docs/thinking-in-react.html
 redirect_from:
   - 'blog/2013/11/05/thinking-in-react.html'
   - 'docs/thinking-in-react-zh-CN.html'
 prev: composition-vs-inheritance.html
 ---
 
-React is, in our opinion, the premier way to build big, fast Web apps with JavaScript. It has scaled very well for us at Facebook and Instagram.
+ React adalah, dalam opini kami, cara paling utama dalam membangun aplikasi besar dengan JavaScript yang cepat. Ia telah men-scale dengan baik untuk kami di Facebook dan Instagram.
```

Comment:
I'm not sure what a proper translation to "scaling" is. Can anyone help?

Reply:
kyaknya `cara paling utama dalam membangun aplikasi besar dengan JavaScript yang cepat.` lebih cocok dengan `cara paling utama dalam membangun aplikasi besar menggunakan JavaScript dengan cepat.` mas.

lalu, untuk kalimat selanjutnya yang `Ia telah men-scale dengan baik untuk kami di Facebook dan Instagram.` mungkin lebih tepat dengan `React telah diskalabilitaskan dengan baik untuk kami di Facebook dan Instagram`.

😄